lib/utils/response.ts

```diff
@@ -13,6 +14,9 @@ import { IResultIterator, iteratorToArray } from "./documentdb";
 export interface IResponseSuccessJsonIterator<T>
   extends IResponse<"IResponseSuccessJsonIterator"> {
   readonly value: T; // needed to discriminate from other T subtypes
+  readonly apply: (
```

Comment:
omg how could it work before?

Reply:
return value is used only in tests (that's why 2 tests fail)